### PR TITLE
Update rust dependencies

### PR DESCRIPTION
**Pull request by jpculp**
**Description of changes:**

Ran `cargo update` for **sources** and **tools**, bumped some crate dependencies, and updated clarify and deny TOMLs as necessary.

**Testing done:**

Built `aws-k8s-1.21` (x86_64 and aarch64). Launched AMIs and ran various host container tests.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: tools/testsys/Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ publish = false
 async-trait = "0.1"
 aws-config = "0.49"
 aws-sdk-ec2 = "0.19"
-base64 = "0.13"
+base64 = "0.20"
```
Comment (jpculp) [inline tools/testsys/Cargo.toml:13]: In general we want to stick with **0.13.x** as there are quite a few crates that locked to **0.13.x**, but `model` has already jumped to **0.20.x**, hence the single bump here.

**Comment by jpculp** [comment]
Rebased and retested.

**Review by webern — APPROVED** [review]

**Review by stmcginnis — APPROVED** [review]
Looks good to me. Would be good to get this in now for some bake time.

**Comment by webern** [comment]
@stmcginnis I'm OK with one of us merging it in @jpculp's absence.

**Pull request merged**